Lib/exception_group.py

```diff
@@ -8,8 +8,11 @@ def __init__(self, excs):
         for e in excs:
             if isinstance(e, ExceptionGroup):
                 for e_ in e.excs:
-                    # TODO: what if e_ is an EG? This looks wrong.
-                    self.tb_next_map[id(e_)] = e_.__traceback__
+                    if isinstance(e_, ExceptionGroup):
+                        for k in e_:
+                            self.tb_next_map[id(k)] = e_.__traceback__
```

Comment:
`for k in e_:` does the recursion - it iterated over individual exceptions.

Reply:
D'oh!

Reply:
Oh, also - I removed the tb_next that you commented on from here. I think we might want to do something in render to avoid duplicates, but let's keep full info here. 

I think what happens is that the frame where an EG is created ends up on both sides of the seam (in `__traceback__` and `__traceback_group__`).